src/hostname/hostnamectl.c

```diff
@@ -5,6 +5,7 @@
 #include <stdbool.h>
 #include <stdlib.h>
 #include <string.h>
+#include <stdint.h>
```

Comment:
sort alphabetically

Reply:
actually, i don't think this line is needed at all